README.md

```diff
@@ -22,6 +22,7 @@ A Python library of exchange calendars meant to be used with [Zipline](https://g
 | New Zealand Exchange            | XNZE     | New Zealand | 1.3           | https://www.nzx.com/                                    |
 | Wiener Borse                    | XWBO     | Austria     | 1.3           | https://www.wienerborse.at/en/                          |
 | Hong Kong Stock Exchange        | XHKG     | Hong Kong   | 1.3           | https://www.hkex.com.hk/?sc_lang=en                     |
+| Irish Stock Exchange            | XDUB     | Ireland     | 1.4*          | http://www.ise.ie/                                      |
```

Comment:
@jmccorriston According to [this](https://en.wikipedia.org/wiki/Euronext_Dublin) the Irish Stock Exchange is now Euronext Dublin. Should I change the name in my PR here?

Reply:
Does Euronext Dublin have a separate iso code? If so, I think we should stick with the Irish Stock Exchange name. If not, we can change it to Euronext Dublin. I'm a little bit worried that FactSet might change the exchange name on their end. What would happen if such a case were to arise?

Reply:
I actually don't think if FactSet changed an exchange name that it would have a consequential effect on us downstream. Euronext Dublin does not appear to have a separate iso code, but that being said I'm going to live this as-is for now and we can revisit it if necessary.